inc/html.class.php

```diff
@@ -3667,6 +3671,10 @@ static function printAjaxPager($title, $start, $numrows, $additional_info = '',
          $back = $start-$list_limit;
       }
 
+      if (!empty($additional_params) && strpos($additional_params, '&') === 0) {
```

Comment:
this part is strange.
I read it as:
- if `$additional_param` string is not empty (strlen better anyway) and if char '&' found at position 0 in this string
- then append, at position 0, the char '&'

is the use case should be prefix the string with the char '&' if not already the case ?

Reply:
Yes, it was what I expected to do. Fixed in commit 1e9ea81c4ba25225ab270bd90f611fa011a52854.